app/controllers/lakeshore/ingest_controller.rb

```diff
@@ -6,11 +6,10 @@ class IngestController < APIController
     # load_and_authorize_resource :curation_concern, class: 'GenericWork'
 
     delegate :intermediate_file, :asset_type, :ingestor, :attributes_for_actor,
-             :check_duplicates?, :represented_resources, :force_preferred_representation?, to: :ingest
+             :check_duplicates_turned_off?, :represented_resources, :force_preferred_representation?, to: :ingest
 
     before_action :validate_ingest, :validate_asset_type, only: [:create]
     before_action :validate_duplicate_upload, :validate_preferred_representations, only: [:create, :update]
-    skip_before_action :validate_duplicate_upload, if: :duplicate_check_param_is_false
```

Comment:
@awead removing this since I added the guard to our `validate_duplicate_upload` before_filter, following your guard clause logic from previous discussions...